app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -103,141 +214,175 @@ class SuggestedEditsTasksFragment : Fragment() {
-        translateDescriptionsTask.imageDrawable = AppCompatResources.getDrawable(requireContext(), R.drawable.ic_icon_translate_title_descriptions)
+    private fun getPageViews(listofSites: ArrayList<WikiSite>) {
+        listofSites.add(WikiSite(Service.COMMONS_URL))
+        listofSites.add(WikiSite(Service.WIKIDATA_URL))
+        val observableList = ArrayList<io.reactivex.Observable<MwQueryResponse>>()
+        for (wikiSite in listofSites) {
+            observableList.add(getPageViewsPerWiki(wikiSite))
+        }
+        disposables.add(io.reactivex.Observable.zip(observableList) { args -> createResult(args) }
+                .subscribeOn(Schedulers.io())
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe({ pageViewsCount ->
+                    pageViewStatsView.setTitle(pageViewsCount.toString())
+                }, L::e))
+    }
 
-        addImageCaptionsTask = SuggestedEditsTask()
-        addImageCaptionsTask.title = getString(R.string.suggested_edits_task_image_caption_title)
-        addImageCaptionsTask.description = getString(R.string.suggested_edits_task_image_caption_description)
-        addImageCaptionsTask.imageDrawable = AppCompatResources.getDrawable(requireContext(), R.drawable.ic_icon_caption_images)
+    private fun createResult(pageViewsResultsList: Array<Any>): Int {
+        var totalPageViews = 0
 
-        translateImageCaptionsTask = SuggestedEditsTask()
-        translateImageCaptionsTask.title = getString(R.string.suggested_edits_task_translate_caption_title)
-        translateImageCaptionsTask.description = getString(R.string.suggested_edits_task_translate_caption_description)
-        translateImageCaptionsTask.imageDrawable = AppCompatResources.getDrawable(requireContext(), R.drawable.ic_icon_caption_translate)
-
-        multilingualTeaserTask = SuggestedEditsTask()
-        multilingualTeaserTask.title = getString(R.string.suggested_edits_task_multilingual_title)
-        multilingualTeaserTask.description = getString(R.string.suggested_edits_task_multilingual_description)
-        multilingualTeaserTask.showImagePlaceholder = false
-        multilingualTeaserTask.showActionLayout = true
-        multilingualTeaserTask.unlockActionPositiveButtonString = getString(R.string.suggested_edits_task_multilingual_positive)
-        multilingualTeaserTask.unlockActionNegativeButtonString = getString(R.string.suggested_edits_task_multilingual_negative)
+        for (result in pageViewsResultsList) {
+            if (result is MwQueryResponse && result.query() != null) {
+                if (result.query()!!.pages() != null) {
+                    for (page in result.query()!!.pages()!!) {
+                        if (page.pageViewsMap != null) {
+                            page.pageViewsMap!!.forEach { (_, value) -> if (value != null) totalPageViews = totalPageViews + value.toInt() }
```

Comment:
Can we change `totalPageViews = totalPageViews + value.toInt()` to `totalPageViews += value.toInt()`